packages/react-storage/src/components/StorageBrowser/actions/handlers/integrity/checkRequiredKeys.ts

```diff
@@ -0,0 +1,17 @@
+export const checkRequiredKeys = <T extends object>(
+  objectToCheck: T,
+  objectName: string,
```

Comment:
Please updates these names to align with function parameter naming within the code base, for example:
```diff
- objectToCheck: T,
+ input: T,
- objectName: string,
+ displayName: string,
```